stream-platform/src/test/java/io/camunda/zeebe/stream/impl/StreamProcessorTest.java

```diff
@@ -466,6 +467,7 @@ public void shouldProcessEvenIfAsyncSchedulingBlocks() throws InterruptedExcepti
   }
 
   @Test
+  @Ignore("Should be enabled when https://github.com/camunda/zeebe/issues/11849 is fixed")
```

Comment:
:x: that's a junit4 extension, I think it's called `Disabled` in JUnit 5

Reply:
Yeah I changed that here [4a18277](https://github.com/camunda/zeebe/pull/11851/commits/4a182772199a84d22a7a6f4f1ba3a16bc3eb1deb)

But somehow intellij is still running the test :cry:

Reply:
Sometimes IntelliJ is confused and caches run configurations. Maybe try running tests in the parent package, that should show you that this test is not run anymore.